_layouts/container-fluid.html

```diff
@@ -3,4 +3,5 @@
 ---
 {% include header.html %}
 {{ page.body | markdownify }}
+</br>
 {% include footer.html %}
```

Comment:
What about wrapping this in some `div` with a utility class? Not really that semantic either but feels slightly more in line with what we are trying to do.

```suggestion
<div class="push-top">{% include footer.html %}</div>
```

Reply:
more context: https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br